sam-react-app/coreconcept/src/counter/Counter.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {Component, createElement} from "react";
+
+class Counter extends Component {
+    state = {
```

Comment:
Classes should always have a constructor and call base constructor, state should be instantiated there. [See the docs](https://legacy.reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class).
Also initial value must be provided as a prop:
> The component should take an initial value in a property.

Reply:
done